### PR TITLE
feat(json-schema-form): make generate_id form editable

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description

- Update `generate_id` form UI to editable ID value

https://user-images.githubusercontent.com/26986739/195266981-cb50b928-5221-4ad6-9b01-9ec2d4cd4c82.mov

### CODE REVIEW

File: src/inputs/forms/new-json-schema-form/components/GenerateIdFormat.vue [hunk 1]
```diff
@@ -53,12 +47,12 @@ export default defineComponent<Props>({
             type: Boolean,
             default: false,
         },
+        invalid: {
+            type: Boolean,
+            default: false,
+        },
     },
     setup(props, { emit }) {
```
Comment (sulmoJ) [inline src/inputs/forms/new-json-schema-form/components/GenerateIdFormat.vue]: Please add "SetupContext"!
```javascript
    setup(props, { emit }: SetupContext) {
```

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: src/inputs/forms/new-json-schema-form/components/GenerateIdFormat.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/inputs/forms/new-json-schema-form/components/GenerateIdFormat.vue]: Thx ~!! 😄

**Pull request merged**